styles/_variables.scss

```diff
@@ -93,3 +69,45 @@ $header__height--min: 80px;
 $header__height--max: 138px;
 $header__padding--max: 16px;
 $header__icon-menu--dimension: 40px;
+
+/* ZENDESK DEFAULTS */
+// Colors
+$brand_color: #0072ef !default;
+$brand_text_color: #fff !default; //Color for text with an accent color background
+$text_color: #333 !default;
+$link_color: #0072ef !default;
```

Comment:
not a blocker, but is there a reason you use `-` for some variables and `_` for others?